db/migrate/20231118123053_create_comments.rb

```diff
@@ -1,7 +1,7 @@
 class CreateComments < ActiveRecord::Migration[7.1]
   def change
     create_table :comments do |t|
-
+      t.string :text
       t.timestamps
     end
   end
```

Comment:
- [ ] Kindly add reference to the post and user table in the comments table to reference the comments relationship with the users and posts in the ERD diagram. Example snippet:

```ruby
      create_table :comments do |t|
    t.text :text
    t.timestamps # this will crate uopdated_at and created_at columns
    t.references :user, foreign_key: true # this will create user_id and a corresponding index  (but students can also add column and index in separate commands)
    t.references :post, foreign_key: true # this will create post_id and a corresponding index (but students can also add column and index in separate commands)
  end


```

Reply:
well resolved! ✔️